gluefactory/models/lines/pold2_extractor.py

```diff
@@ -322,6 +322,8 @@ def filter_with_distance_field(
         points_coordinates = self.get_coordinates(
             points, indices, self.coeffs_list[sample_idx], self.coeffs_second_list[sample_idx]
         )
+        points_coordinates[:, 0] = torch.clamp(points_coordinates[:, 0], 0, distance_map.shape[1] - 1)
```

Comment:
Why do we need the clamping here?

Reply:
point_coordintates in rdnim sometimes fell exactly on the border, which led to indexing errors when indexing the df.